src/pages/group/groupqueries.ts

```diff
@@ -26,11 +26,8 @@ export const useCreatePost = (title: string, content: string, groupID: number, u
     
 })
 
-export const useRemoveMember = (selectedUserId: number, groupId: number) => useQuery< void , Error>({
-    queryKey: ["removemember", selectedUserId, groupId],
-    queryFn: () => removeMember(selectedUserId, groupId),
-    
-})
+//export const useRemoveMember = (selectedUserId: number, groupId: number) => 
+//useMutation
```

Comment:
If u can't do something leave the working code still. So here need to return the old part